README.md

```diff
@@ -4,53 +4,75 @@ Shell script alias manager.
 
 ## Getting Started
 
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. 
 
 ### Prerequisites
 
-What things you need to install the software and how to install them
-
-```
-Give examples
-```
+Ali requirements for development are Go and a unix based system.
 
 ### Installing
 
-A step by step series of examples that tell you how to get a development env running
-
-Say what the step will be
+Clone the repository in your computer:
 
 ```
-Give the example
+$ git clone git@github.com:adriacidre/ali.git
 ```
 
-And repeat
+Then set up your ./basrc or ./zshrc with some commands like the ones below with your own $GOPATH
+
+```
+[ -f ~/.aliases ] && source ~/.aliases
+ali() {
```

Comment:
@litosnano can you create an issue so this alias `ali` is managed internally by this library instead of having to manually add it to the `zshrc` please?

Reply:
Issue created